### DISCUSSION TITLE
Looking for maintainers

### DISCUSSION THREAD

**Discussion by SteffeyDev** [SteffeyDev/react-native-popover-view#163] [Announcements]
I no longer have any React Native apps that I actively maintain, and so am thus detached from the community and need to focus my attention elsewhere. If you actively use this project and are interested in maintaining it, please reach out and let me know!

**Comment by AdamGerthel** [comment]
I might be able to take over, because I'm currently looking into https://github.com/SteffeyDev/react-native-popover-view/issues/37 and will likely be making a PR soon.

BTW - I noticed that the project isn't using react-native-builder-bob or similar tools. How do you usually test your changes locally during development? RN doesn't support symlinks, for instance.

**Reply by SteffeyDev** [reply]
I created https://github.com/SteffeyDev/react-native-popover-view-test-app for testing. That repo has a `transfer` script, written for my specific system and folder structure, that copies react-native-popover-view into `node_modules` of the test app so that I can test the changes.

**Reply by SteffeyDev** [reply]
So, `yarn watch` in the `react-native-popover-view` folder to continuously generate the `dist` folder, then I did `./transfer` after every change to pull the changes into the test app repo for testing.

**Reply by SteffeyDev** [reply]
I'd be happy to add you as a maintainer, and then eventually transfer the repo if you stick around.

**Reply by AdamGerthel** [reply]
I'll fix the issue first, then we'll see :D

Have the test app up and running now on latest Expo SDK, and using [fswatch](https://github.com/emcrisostomo/fswatch) to simplify the transfer.

**Reply by AdamGerthel** [reply]
PR opened!

**Comment by AdamGerthel** [comment]
@SteffeyDev A couple of issues have arisen with the new arch. I'd be happy to review the PR's and issues and help solve it if you can give me the required permissions

**Reply by SteffeyDev** [reply]
That would be great, I've invited you

**Reply by SteffeyDev** [reply]
What is your NPM username?

**Reply by AdamGerthel** [reply]
> What is your NPM username?

insats

**Reply by AdamGerthel** [reply]
Could you give me contributor access to https://github.com/SteffeyDev/react-native-popover-view-test-app as well?

**Reply by SteffeyDev** [reply]
Yes, I've invited you there as well
